src/main/java/duke/commands/ByeCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.commands;
+
+import duke.data.TaskList;
```

Comment:
I like how the import statements are explicitly specified rather than a wildcard import statement.